frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -43,6 +55,38 @@ pub struct SignedSubmission<AccountId, Balance: HasCompact, CompactSolution> {
 	pub solution: RawSolution<CompactSolution>,
 }
 
+impl<AccountId, Balance, CompactSolution> Ord
```

Comment:
would be better for readability if you put the impl after the thing is declared.